package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "shaka-player",
   "description": "DASH/EME video player library",
-  "version": "4.3.0",
+  "version": "4.4.0",
```

Comment:
Why?

Reply:
I don't know how this happened. I'll revert.